.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,69 @@
+name: Bug report
+description: Submit a bug report
+labels: ["type-bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **New to Python?**
+
+        For help or advice on using Python, try one of the following options instead of opening a GitHub issue:
+
+          - Posting on [Discourse](https://discuss.python.org/c/users/7)
+          - Reading the [Python tutorial](https://docs.python.org/3/tutorial/)
+          - Emailing [python-list](https://mail.python.org/mailman/listinfo/python-list)
+  - type: checkboxes
+    attributes:
```

Comment:
I like these checkboxes, I'm wondering if they need to be required? 

For example, we really want people to report potential (release blocker) bugs in pre-releases, even if they're not 100% confident it's a CPython bug.

Although I can definitely see why they are required, to help avoid all those reports to the wrong repo, and duplicates.

Reply:
That's a good point. Let's make them "not-required" for now; we can always change them to "required" later if people still aren't paying them any notice when filing issues.